pkg/operation/botanist/component/extensions/operatingsystemconfig/original/components/kubelet/config.go

```diff
@@ -147,6 +147,11 @@ var (
 	}
 )
 
+func ProtectedKernelDefaultsEnabled(kubeletConfigParameters components.ConfigurableKubeletConfigParameters, kubernetesVersion *semver.Version) bool {
```

Comment:
Please add a doc string.

I'd also rather see this utility in `pkg/apis/core/v1beta1/helper/helper.go`.

Reply:
Having in mind the result of [pull-gardener-unit](https://prow.gardener.cloud/view/gs/gardener-prow/pr-logs/pull/gardener_gardener/7088/pull-gardener-unit/1597882096603369472) job I think it is logical that we do not import `pkg/operation/botanist/component/extensions/operatingsystemconfig/original/components` in `pkg/apis/core`. Maybe it will be best if we leave this helper function under `pkg/operation/botanist/component/extensions/operatingsystemconfig/original/components/kubelet` since it is either way tightly coupled to the kubelet and only relevant for it. WDYT @timuthy ?

Reply:
Yes, makes sense 👍 My bad, I thought we operate directly on the API object.